governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Template - [Executive Vote] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023
+summary: Scope-defined parameter changes; Spark Protocol D3M parameter changes; set Fortunafi Debt Ceiling to zero; approve DAO Resolution for HV Bank to terminate future purchases; Aligned Delegate compensation for August 2023; trigger Spark Protocol Proxy Spell; Core Unit MKR vesting transfers; scuttle MCD_CAT contract.  
```

Comment:
The summary should contain more information than the title. Although it's a lengthy summary, it should still provide valuable details. Here are some suggestions:

- Specify which parameter fields are being modified.
- Combine information about Spark changes to encompass alterations to Spark D3M and spell execution.
- Remove "approve" from HBV.
- Include MKR expenditures.
- Add housekeeping items.

Reply:
The summary can be more a matter of personal preference, I think.

As for my summary here, I wrote this in accordance with how I did things during our training, and it also aligns with previous Exec vote examples. The summary lists all items, versus just the higher-impact ones, but I personally don't think the summary should be packed full of details.

Reply:
The issue is that presently, the summary isn't providing much value as it merely repeats the title (often verbatim). It's not a critical problem, but it ties into my other feedback about adding unnecessary words and length to the spell without enhancing its utility for MKR voters.